### PR TITLE
(feat) : Add payment status column on bill overview and payment history tables

### PR DESCRIPTION
**Pull request by Ogollah**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

#### What does this PR do?
* Add Status in the payment history table
* Add Status in the Overview patient bill summary

## Screenshots
Screenshot from 2024-11-10 22-53-10

Screenshot from 2024-11-11 10-02-50

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const headers = [
   { header: 'Total Amount', key: 'totalAmount' },
   { header: 'Service', key: 'billingService' },
   { header: 'Reference Codes', key: 'referenceCodes' },
+  { header: 'Status', key: 'status' },
```
Comment (makombe) [inline packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx:35]:
```suggestion
  { header:t('status', 'Status'), key: 'status' },
```

**Review by makombe — APPROVED** [review]
Thanks @Ogollah LGTM. Just a small suggestion

File: packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx (hunk 1 above)
Comment (ojwanganto) [reply packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx:35]: It is like all the header labels have not been translated. Should that be done in this PR or can be in a followup PR?

Comment (Ogollah) [reply packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx:35]: I can do the translation in a different PR. What do you think @makombe

Comment (donaldkibet) [reply packages/esm-billing-app/src/billable-services/payment-history/payment-history-viewer.component.tsx:35]: Thanks @Ogollah

**Pull request merged**